### DISCUSSION TITLE
Scope the "!important" flag to a specific set of classes

### DISCUSSION THREAD

**Discussion by vortechron** [tailwindlabs/tailwindcss#1645] [Help]
ive enabled important feature: 

tailwind.config.js
```
module.exports = {
  important: true,
}
```

but dont want certain class like .flex to have !important on it. how can i do this?

**Comment by AhnanWong** [comment]
I have the same problem...

**Comment by gomezcabo** [comment]
Me too. I need to rethink every v-show on my Vue project. It would be nice to have a way to avoid important in some utilities.
